source/renderer/zettlr-editor.js

```diff
@@ -447,30 +453,35 @@ class ZettlrEditor {
     this._cm.on('renderLine', (cm, line, elt) => {
       let monospaceWidth = this.computeMonospaceWidth()
       let spaceWidth = this.computeSpaceWidth()
+      let tabWidth = spaceWidth * 5
       let basePadding = 2.5 * monospaceWidth
 
       // Show continued list/quote lines aligned to start of text rather than first non-space char (hanging indent)
       // MINOR BUG: also does this inside literal blocks.
```

Comment:
I think we might drop that comment, as I think I fixed that with my last PR (I don't think that your changes re-introduced that, but we might double-check!)